recipes/vdom/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "vdom" %}
+{% set version = "0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d43fb8e92e64e1cabbdb93b4f213b95145db0e843371239b3a8bf91dbdf65159"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - ipython
+    - jsonschema
+    - pip
+    - python
+  run:
+    - ipython
+    - jsonschema
+    - python
+
+test:
+  imports:
+    - vdom
+
+about:
+  home: "https://github.com/nteract/vdom"
+  license: "BSD"
```

This should specify which varient of the BSD licence applies:
```suggestion
  license: BSD-3-Clause
```